docs/pages/setup/admin/trustedclusters.mdx

```diff
@@ -277,11 +371,33 @@ spec:
     - remote: admin
       # admin <-> admin works for the Open Source Edition. Enterprise users
       # have great control over RBAC.
-      local: [access]
+      local: [admin]
   token: "join-token-from-root"
   tunnel_addr: root.example.com:3024
   web_proxy_addr: root.example.com:3080
 ```
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+```yaml
+# Save this as root-cluster.yaml on the auth server of "leaf" and then execute:
```

Comment:
This whole section seems technically correct but when viewing the file in github, is not formatted properly like the section above it.  Instead, what is supposed to look like yaml is jumbled together as a paragraph without line breaks in the proper places.

Reply:
Does this look okay if you view the docs site locally with these steps?
- Open a clone of `gravitational/next`
- Run `git submodule update --init --recursive`
- Navigate to `content/9.0`
- Check out my branch
- Run `yarn dev`
- Open the page

Reply:
Yes, yes it does.